src/core/lib/transport/error_utils.cc

```diff
@@ -42,11 +43,59 @@ static grpc_error_handle recursively_find_error_with_field(
   return absl::OkStatus();
 }
 
+namespace {
+
+std::optional<grpc_http2_error_code> GetHttp2Error(const absl::Status& status) {
```

Comment:
Please add a TODO for @tanvi-jagtap to make this go away (she's building a HTTP/2 error type)

Reply:
Added the TODO in status_helpers.h, where the property is defined.